src/libPersistence/BlockStorage.cpp

```diff
@@ -133,6 +133,20 @@ bool BlockStorage::GetTxnFromHistoricalDB(const dev::h256& key,
   return true;
 }
 
+bool BlockStorage::GetHistoricalMicroBlock(const BlockHash& blockhash,
```

Comment:
Is there supposed to be a `PutHistoricalMicroBlock`, or any code that stores into `m_MBHistoricalDB`?

Reply:
actually that is done by backend using s3